### PR TITLE
2016-01-26-segwit-benefits.md

### PR DESCRIPTION
**Pull request by friendsofbitcoin**
2016-01-26-segwit-benefits.md  simplified chinese zh_CN, please suggest only simplified chinese corrections

### CODE REVIEW

File: _posts/zh_CN/posts/2016-01-26-segwit-benefits.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+未来hardfork如能改变单一容量限制，使用者就能受惠。 如：
+
+    50* sigops + 4*basedata + 1*witnessdata < 10M
+
+使用者能轻易准确地填写单一数据同时最大限度并提高手续费收入，这让他们更可靠地计算出需要进行开采其交易的费用让用户受益。
+
+### 更多资料
+
+ * [背包问题](https://en.wikipedia.org/wiki/Knapsack_problem)
+ * [2015 年 8 月 Sigop 攻击 bitcoin 的讨论](https://bitcointalk.org/index.php?topic=1166928.0;all)
```
Comment (shangzhou) [inline _posts/zh_CN/posts/2016-01-26-segwit-benefits.md]: 应该是bitcointalk

File: _posts/zh_CN/posts/2016-01-26-segwit-benefits.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+ * [Gregory Maxwell 对比特币-dev证人的限制 ](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-December/011870.html)
```
Comment (shangzhou) [inline _posts/zh_CN/posts/2016-01-26-segwit-benefits.md]: 我觉得这里就直接“bitcoin-dev邮件列表”比较好

Comment (friendsofbitcoin) [inline _posts/zh_CN/posts/2016-01-26-segwit-benefits.md]: thnx , done

Comment (shangzhou) [inline _posts/zh_CN/posts/2016-01-26-segwit-benefits.md]: huh? i mean "比特币-dev" should be "bitcoin-dev 邮件列表"

Comment (friendsofbitcoin) [inline _posts/zh_CN/posts/2016-01-26-segwit-benefits.md]: sorry , I don't speak manadarin , just paying for translation as no one else is translating these documents . Thus the confusion. Thanks for clarifying. Updated

Comment (shangzhou) [inline _posts/zh_CN/posts/2016-01-26-segwit-benefits.md]: but still confusing, i think better translation is like "Gregory Maxwell 在bitcoin-dev 邮件列表上发表的witness limits" or do nothing about this.

**Comment by btcdrak** [comment]
ping @jl2012

**Pull request merged**